tests.ts

```diff
@@ -216,4 +228,28 @@ ClearBlade.Trigger.Fetch("triggername", genericCallback);
 ClearBlade.Timer.Create("timername", {}, genericCallback);
 ClearBlade.Timer.Fetch("timername", genericCallback);
 
+///////////////////////////////////////
+// Resp API
+///////////////////////////////////////
+const customResponseCallBack: CbServer.CbCallback = (
```

Comment:
Are these consts that are of type CbServer.CbCallback meant to represent a code service entry point? If so, this type isn't right since the first parameter is `BasicReq`